### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by htorrence**
Adding a note about the name of the project.

Ended up also fixing some linting stuff + adding piptools to make our CI happy.

Pull Request Checklist
 - [x] Pull request is based on WIP branch
 - [x] Pull request includes a description of the change and the reason behind it
 - [x] `pytest` passes locally
 - [x] `flake8 apparate tests` passes locally
 - [x] CHANGELOG has been updated
 - [x] Version in `_version.py` has been updated
 - [x] README and docs have been updated (if applicable)

### CODE REVIEW

File: requirements-dev.in [hunk 1]
```diff
@@ -0,0 +1 @@
+-e file:.[dev]
```
Comment (sugi-chan) [inline requirements-dev.in:1]: does this need a newline?

File: tests/test_cli.py [hunk 2]
```diff
@@ -2,7 +2,6 @@
 from os.path import expanduser, join
 from unittest import mock
 
-import pytest
```
Comment (sugi-chan) [inline tests/test_cli.py:5]: we dont need pytest things anymore? fancy?

File: README.md [hunk 3]
```diff
@@ -1,8 +1,7 @@
 # Apparate
 Make your libraries magically appear in Databricks.
 
-![apparate](https://pa1.narvii.com/6547/b8fd048a32aa591dbae528f4a80e67cb87112317_hq.gif)
-
+> **Note 6/15/20:** Our team previously had a tradition of naming projects with terms or characters from the Harry Potter series, but we are disappointed by J.K. Rowling’s persistent transphobic comments. In response, we will be renaming this repository, and are working to develop an inclusive solution that minimizes disruption to our users.
```
Comment (sugi-chan) [inline README.md:4]: yasssss

File: tests/test_cli.py (hunk 2 above)
Comment (htorrence) [reply tests/test_cli.py:5]: yeah honestly I'm not 100% sure what these marks did in the first place, but my best guess is that they are an outdated format. Pytest was throwing warnings about them and the tests all pass without them.

**Pull request merged**